HelpSource/Classes/Pclump.schelp

```diff
@@ -0,0 +1,38 @@
+TITLE:: Pclump
+summary:: Clumps the pattern into groups of user-defined sizes.
```

Comment:
could you take anothe look at what I suggested? The pattern isn't clumped itself. 

Also "user-defined" is a little redundant: just say "into groups of a given size"

Reply:
thanks for your work, almost there! :)

Reply:
Oh sorry, I missed that. I made a slight change to the description as well @telephon. Is this better?